src/memspace.c

```diff
@@ -426,3 +426,133 @@ umfMemspaceMemtargetRemove(umf_memspace_handle_t hMemspace,
     hMemspace->size--;
     return UMF_RESULT_SUCCESS;
 }
+
+static int MemspaceFilter(umf_memspace_handle_t memspace,
```

Comment:
shouldn't it be called "umfMemspaceFilter"?

Reply:
done